src/pages/dmt5/_margin_calculator.js

```diff
@@ -0,0 +1,276 @@
+import React from 'react'
+import Proptypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import styled from 'styled-components'
+import { isMobile } from 'common/os-detect'
+import { isBrowser } from 'common/utility'
+import { Box, Flex, SectionContainer } from 'components/containers'
+import { Carousel, Header, LinkText, QueryImage, Text } from 'components/elements'
+import { LinkButton } from 'components/form'
+import { Localize } from 'components/localization'
+import device from 'themes/device'
+
+const query = graphql`
```

Comment:
Can you check up with the design team regarding the images of the calculators, because they're different with the current ready calculators 

https://app.zeplin.io/project/5e018df9bc188b18338426df/screen/5f599c97c0b1eb436868c016